less/Vetrina.less

```diff
@@ -98,6 +98,10 @@
     }
 }
 
+triggerbee-widget {
```

Comment:
I guess this should be `.triggerbee-widget` (dot in front)?

Reply:
<img width="257" alt="image" src="https://user-images.githubusercontent.com/60072001/234228771-553882ff-d604-4e3e-b784-c9ff7fdd2b32.png">
Well, it's an element called that, not an element with that class

Reply:
Ah, that's a weird tag name. Yeah, the code is correct then.